File: sdks/Unity/Breeze/Plugins/Android/breeze.androidlib/src/com/breeze/sdk/BreezeConstants.java
package com.breeze.sdk;

public final class BreezeConstants {

    private BreezeConstants() {}

    public static final String SDK_VERSION = "0.1.0";
    public static final String SDK_PLATFORM = "android";

    /** Domains allowed for webview navigation and JS bridge injection. */
    public static final String[] ALLOWED_HOSTS = {".breeze.cash", ".breeze.com"};

    /** Custom-scheme payment redirect constants. */
    public static final String PAYMENT_REDIRECT_HOST = "breeze-payment";
    public static final String PAYMENT_REDIRECT_SUCCESS_PATH = "/purchase/success";
    public static final String PAYMENT_REDIRECT_FAILURE_PATH = "/purchase/failure";

    /** JavaScript interface name exposed to the web page. */
    public static final String JS_INTERFACE_NAME = "_breezeNative";
}

File: sdks/Unity/Breeze/Plugins/iOS/BreezeConstants.swift
import Foundation

enum BreezeConstants {
    static let sdkVersion = "0.1.0"
    static let sdkPlatform = "ios"

    /// Domains allowed for webview navigation and JS bridge injection.
    static let allowedHosts = [".breeze.cash", ".breeze.com"]

    /// Host and paths for custom-scheme payment redirects.
    enum PaymentRedirect {
        static let host = "breeze-payment"
        static let successPath = "/purchase/success"
        static let failurePath = "/purchase/failure"
    }

    /// WKScriptMessageHandler names (must match JS bridge property names).
    enum MessageHandler {
        static let onPaymentSuccess = "onPaymentSuccess"
        static let onPaymentFailure = "onPaymentFailure"
        static let dismiss = "dismiss"
        static let getDeviceInfo = "getDeviceInfo"

        static let all = [onPaymentSuccess, onPaymentFailure, dismiss, getDeviceInfo]
    }
}
